language/phpmailer.lang-mg.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+* PHPMailer language file: refer to English translation for definitive list
+* French Version
```

Comment:
i'm sure this file is not for french

Reply:
@Synchro Can't we put Google translations for the missing ones till someone actually translate it?

Reply:
Sure - apparently nobody is particularly bothered about this translation, so you could fork it and resubmit the PR.